# HG changeset patch
# User Cole Robinson <crobinso@redhat.com>
# Date 1254247348 14400
# Node ID 59f3d7a2ec289145c41437b7f12bcd730e17f340
# Parent  6d219a6db4fbcb343d41a959dac7a8623f3dcbb3
manager: Actually update VM state text when state changes

diff -r 6d219a6db4fb -r 59f3d7a2ec28 src/virtManager/manager.py
--- a/src/virtManager/manager.py	Tue Sep 29 14:00:50 2009 -0400
+++ b/src/virtManager/manager.py	Tue Sep 29 14:02:28 2009 -0400
@@ -584,6 +584,7 @@
         row[ROW_STATUS] = vm.run_status()
         row[ROW_STATUS_ICON] = vm.run_status_icon_large()
         row[ROW_IS_VM_RUNNING] = vm.is_active()
+        row[ROW_MARKUP] = self._build_vm_markup(vm, row)
         model.row_changed(row.path, row.iter)
 
 
